src/main/java/io/vlingo/actors/plugin/mailbox/telemetry/TelemetryMailbox.java

```diff
@@ -0,0 +1,66 @@
+package io.vlingo.actors.plugin.mailbox.telemetry;
+
+import io.vlingo.actors.Mailbox;
+import io.vlingo.actors.Message;
+
+public class TelemetryMailbox implements Mailbox {
+  private final MailboxTelemetry telemetry;
+  private final Mailbox delegate;
```

Comment:
Is the real mailbox truly a delegate? Maybe to the telemetry, but I might name it "real" or "actual". I suppose their could be a telemetry inside a telemetry, but I kinda hope not :)

Reply:
Telemetry of the telemetry will be amazing. But I can change the name to 'real' though I don't really like the name. Maybe 'backend' is better?

Reply:
Never mind. Leave it as delegate :)